docs/USER_GUIDE.md

```diff
@@ -42,12 +42,15 @@ The Fargate tasks used by this solution to perform deletions must be able to
 access the following AWS services, either via an Internet Gateway or via [VPC
 Endpoints]:
 
-- Amazon S3
-- Amazon DynamoDB
-- Amazon CloudWatch (monitoring and logs)
-- AWS ECR
-- Amazon SQS
-- AWS STS
+- Amazon S3 gateway endpoint (_com.amazonaws.**region**.s3_)
```

Comment:
the above sentence no longer makes sense with this updated list
> The Fargate tasks used by this solution to perform deletions must be able to
access the following AWS services, either via an Internet Gateway or via [VPC
Endpoints]:

The new list is a list of endpoints rather than services

Reply:
You're right - updated, better?